src/ui/components/ServersView/DocumentViewer.tsx

```
@@ -89,13 +89,68 @@ const DocumentViewer = ({
           display='flex'
           color={theme === 'dark' ? 'font-white' : 'font-text'}
         >
-          <IconButton
-            icon='arrow-back'
-            onClick={closeDocumentViewer}
-            mi='x8'
-            color={theme === 'dark' ? 'white' : 'default'}
-          />
+
+           <IconButton
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)